### PR TITLE
UI: Add footer text option for login screen

### PR DESCRIPTION
**Pull request by utchoang**
### Description

This PR fixes https://github.com/apache/cloudstack/issues/5622

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by utchoang** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]

File: ui/public/config.json [hunk 1]
```diff
@@ -56,5 +56,6 @@
   "plugins": [],
   "basicZoneEnabled": true,
   "multipleServer": false,
-  "docHelpMappings": {}
+  "docHelpMappings": {},
```
Comment (rhtyd) [inline ui/public/config.json:59]: @shwstppr does this break the automatic addition of doc help mappings?

File: ui/public/config.json [hunk 2]
```diff
@@ -56,5 +56,6 @@
   "plugins": [],
   "basicZoneEnabled": true,
   "multipleServer": false,
+  "loginFooter": "",
```
Comment (PaulAngus) [inline ui/public/config.json:59]: Isn't up with the main ui footer text, a more obvious place to put this setting in the config.json file?

```
  "docBase": "http://docs.cloudstack.apache.org/en/latest",
  "appTitle": "CloudStack",
  "footer": "Licensed under the <a href='http://www.apache.org/licenses/' target='_blank'>Apache License</a>, Version 2.0.",
  "loginFooter": "",
  "logo": "assets/logo.svg",
  
```

**Comment by rhtyd** [comment]
@blueorangutan ui

**Comment by sureshanaparti** [comment]
@blueorangutan ui

**Pull request merged**